migrations_lockfile.txt

```diff
@@ -8,10 +8,9 @@ will then be regenerated, and you should be able to merge without conflicts.
 admin: 0002_logentry_remove_auto_add
 auth: 0008_alter_user_username_max_length
 contenttypes: 0002_remove_content_type_name
-demo: 0002_demo_org_initializing_state
```

Comment:
Running `sentry django makemigrations sentry --empty -n <name>` removed this

Reply:
@scefali feels like we need this? Any idea why it would be removed like this?

Reply:
@silent1mezzo good catch, we definitely need this. I have no idea why this would happen, maybe @wedamija knows

Reply:
It's because we run `sentry django makemigrations` with our default config, which does not install the `demo` app. This isn't super important for `demo` at the moment - it's mostly there to prevent migration conflicts within an app on master, and since Steve is the only one working on it, those are less likely.

Not sure what the best option for this is going forward. Maybe we can figure out how to have a separate lockfile just for demo.

Reply:
@shruthilayaj could you just manually edit the file to leave the line in?

Reply:
For the moment it might be better to just leave `demo` out of here... otherwise everyone making migrations is going to need to know about this.